test/crowdsale/RefundableCrowdsale.test.js

```diff
@@ -30,56 +30,86 @@ contract('RefundableCrowdsale', function ([_, owner, wallet, investor, purchaser
     this.openingTime = (await latestTime()) + duration.weeks(1);
     this.closingTime = this.openingTime + duration.weeks(1);
     this.afterClosingTime = this.closingTime + duration.seconds(1);
+    this.preWalletBalance = await ethGetBalance(wallet);
 
     this.token = await SimpleToken.new();
-    this.crowdsale = await RefundableCrowdsale.new(
-      this.openingTime, this.closingTime, rate, wallet, this.token.address, goal, { from: owner }
+  });
+
+  it('it requires a non-zero goal', async function () {
+    await expectThrow(
+      RefundableCrowdsale.new(
+        this.openingTime, this.closingTime, rate, wallet, this.token.address, 0, { from: owner }
```

Comment:
These tests would be a lot clearer if we had a helper like `newRefundableCrowdsale({ goal: 0})` that just used the default values for everything else.